nengo_dl/converter.py

```diff
@@ -627,12 +627,7 @@ def get_input_obj(self, node_id, tensor_idx=0, allow_none=True):
 
         input_layer, input_node_id, input_tensor_id = self.get_history(tensor)
 
-        if input_node_id in self.converter.layer_map[input_layer]:
-            return self.converter.layer_map[input_layer][input_node_id][input_tensor_id]
-        elif allow_none:
-            return None
-        else:
-            raise ValueError("Input tensor %s has not been converted" % (tensor))
+        return self.converter.layer_map[input_layer][input_node_id][input_tensor_id]
```

Comment:
It was my impression that sometimes this function needed to be called on nodes that aren't in the layer map, and return `None` in that case to indicate that. But if that's not necessary, this is certainly much simpler!

Reply:
That only happens in one case, in `ConvertInputs`, so I just changed it to directly handle that case in that function, rather than handling it in this generic method.